src/main/department/Department.java

```diff
@@ -0,0 +1,52 @@
+package main.department;
+
+
+import main.member.Members;
+
+import java.util.List;
+
+public abstract class Department {
+    protected String nameDepartment;
+    protected static final int NR_MAX_VOLUNTEER = 10;
+    protected List<Members> listMembers;
+
+    public Department(String nameDepartment, List<Members> listMembers) {
+        this.nameDepartment = nameDepartment;
+        this.listMembers = listMembers;
+    }
+
+    public boolean addMemberToDepartment(Members member) {
+        int actualSize = listMembers.size();
+        if (actualSize < NR_MAX_VOLUNTEER) {
+            listMembers.add(member);
+            return true;
+        } else if (actualSize == NR_MAX_VOLUNTEER) {
```

Comment:
You don't need 2 else branches just to show a different message. Use just a simple else with a generic message for both cases (== and >).

Reply:
Also, in Model classes you should not "print" anything.